### ISSUE TITLE
There is no Autosplitter for this game

### ISSUE THREAD

**Issue by FireCulex** [SuiMachine/LiveSplit.FEAR#1]
https://i.imgur.com/SvGnEak.png

Livesplit 1.7.6

**Comment by SuiMachine** [comment]
True. I have probably never added it, because of the issues it's been having at the beginning. Does it work tho if you add it manually? If so, I'll add it to XML.

**Comment by FireCulex** [comment]
Yes, My only critique however is the timer stops when the loading screen displays, which is correct, but starts again when the "Press Any Key" is displayed as opposed to after you have pressed the key and started the level.  Does that make sense? If I dont press anything and sit at the loading screen indefinitely the timer will just continue counting. Don't know if theres a way to adjust it, its just a minor thing, otherwise it works just fine.

**Comment by SuiMachine** [comment]
Yeah, there is some discrepancy between when the load actually ends and game play begins. Since first version was very much unreliable, with the updated one (which I made already 2 years ago... time flies), I went with the most reliable thing I was able to find for updated version, but again - didn't want to over correct with gameplay/renderer start etc. Partially because I'm lazy, partially because if I were to really flatten it completely, I'd also have to look into removing save times and then it's getting a bit absurd. Idk, maybe one day, when I feel great with reverse engineering and FEAR community times will be down to millisecond difference, I'll look into correcting that.

As for LiveSplit Autosplitter XML, I modified it and sent pull request:
https://github.com/LiveSplit/LiveSplit/pull/1160
